README.md

```diff
@@ -111,6 +111,19 @@ If you see a bounty that you think is particularly important you can upvote it t
 
 <br>
 
+# Thanks For Contributing
+XRPL Bounties would like to express our sincere gratitude to all of the contributors who have helped to improve the XRPL ecosystem through their participation in bounties. Your hard work and dedication are highly valued and appreicated. 
+## Contributor list (No Particular Order):
+- [kasiimh1](https://github.com/kasiimh1)
```

Comment:
What about just putting it in alphabetical order by username?

Reply:
Agreed and the list shouldn't be based on the number of milestones and what not, just alphabetical order to show gratitude to all contributors 👍🏼